air-quality-backend/database_tests/utils/CAMS_forecast_tests.py

```diff
@@ -0,0 +1,44 @@
+from database_tests.utils.db_helpers import get_database_data
+from dotenv import load_dotenv
+
+load_dotenv()
+
+collection_name = "forecast_data"
+
+
+def test_aqi_levels_are_between_1_and_6():
+    query = {}
+    dict_result = get_database_data(query, collection_name)
+    pollutant_keys = ["no2_aqi_level", "so2_aqi_level", "o3_aqi_level",
+                      "pm10_aqi_level", "pm2_5_aqi_level"]
+
+    for document in dict_result:
+        overall_aqi_level = document["overall_aqi_level"]
+        assert 1 <= overall_aqi_level <= 6, f" {overall_aqi_level} is out of range"
+
+    for document in dict_result:
+        for key in pollutant_keys:
+            assert 1 <= document[key] <= 6, f"{key} {document[key]} is out of range"
```

Comment:
I think this should be two tests, it's better to have only one reason for a test to fail

Reply:
Sure thing, I did debate on whether to have them separately 🤔